Core/Libraries/Source/WWVegas/WWLib/wwstring.h

```diff
@@ -553,19 +553,19 @@ StringClass::Get_Buffer (int new_length)
 }
 
 ///////////////////////////////////////////////////////////////////
-//	Peek_Buffer
+//	str (formerly Peek_Buffer)
 ///////////////////////////////////////////////////////////////////
 inline TCHAR *
-StringClass::Peek_Buffer (void)
+StringClass::str (void)
```

Comment:
I think this is quite a bit confusing in the context of the GameEngine, because `AsciiString::str` returns `const char*`.